config/authorization/config.ex

```diff
@@ -155,21 +198,35 @@ defmodule Acl.UserGroups.Config do
                       ] } } ] },
 
       # // LOKETADMIN
-        %GroupSpec{
-          name: "o-admin-rwf",
-          useage: [:read, :write, :read_for_write],
-          access: access_by_role( "LoketAdmin" ),
-          graphs: [ %GraphSpec{
-                      graph: "http://mu.semte.ch/graphs/organizations/",
-                      constraint: %ResourceConstraint{
-                        resource_types: [
-                          "http://lblod.data.gift/vocabularies/reporting/Report",
-                          "http://vocab.deri.ie/cogs#Job",
-                          "http://open-services.net/ns/core#Error",
-                          "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#DataContainer",
-                          "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#FileDataObject",
-                          "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#DataContainer"
-                        ] } } ] },
+      %GroupSpec{
+        name: "o-admin-sessions-rwf",
+        useage: [:read, :write, :read_for_write],
+        access: is_admin(),
+        graphs: [
+          %GraphSpec{
+            graph: "http://mu.semte.ch/graphs/sessions",
+            constraint: %ResourceFormatConstraint{
+              resource_prefix: "http://mu.semte.ch/sessions/"
+            }
+          },
+        ]
+      },
+
+      %GroupSpec{
+        name: "o-admin-rwf",
+        useage: [:read, :write, :read_for_write],
+        access: access_by_role( "LoketAdmin" ),
```

Comment:
Nele should be able to give the actual role if you don't have access yet to check it that way.